pkg/storage/replica.go

```diff
@@ -3548,6 +3543,27 @@ func (r *Replica) tickRaftMuLocked() (bool, error) {
 	return true, nil
 }
 
+// maybeTickQuiesced attempts to tick a quiesced or dormant replica, returning
+// true on success and false if the regular tick path must be taken
+// (i.e. Replica.tick).
+func (r *Replica) maybeTickQuiesced() bool {
+	var done bool
+	r.mu.Lock()
+	if r.mu.internalRaftGroup == nil {
+		done = true
+	} else if r.mu.quiescent {
+		done = true
+		if tickQuiesced {
+			// NB: It is safe to call TickQuiesced without holding Replica.raftMu
```

Comment:
You couldn't have two goroutines trying to tick the replica at the same time and thus clobber the counter in undefined ways?